src/chains/opStack/errors/withdrawal.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseError } from '../../../errors/base.js'
+import type { Hex } from '../../../types/misc.js'
+
+export type ReceiptContainsNoWithdrawalsErrorType =
+  ReceiptContainsNoWithdrawalsError & {
+    name: 'ReceiptContainsNoWithdrawalsError'
+  }
+export class ReceiptContainsNoWithdrawalsError extends BaseError {
+  override name = 'ReceiptContainsNoWithdrawalsError'
```

Comment:
```suggestion
  override readonly name = 'ReceiptContainsNoWithdrawalsError'
```
```suggestion
  override name: 'ReceiptContainsNoWithdrawalsError' = 'ReceiptContainsNoWithdrawalsError'
```
Oh I see I'm guessing name is of type string if we don't do one of these